packages/fast-viewer-react/tsconfig.json

```diff
@@ -1,7 +1,28 @@
 {
     "compilerOptions": {
+        "baseUrl": ".",
+        "emitDecoratorMetadata": true,
+        "experimentalDecorators": true,
+        "declaration": true,
+        "lib": [
+            "es2015",
+            "es2017",
+            "dom",
+            "scripthost"
+        ],
+        "jsx": "react",
+        "module": "CommonJS",
+        "moduleResolution": "node",
+        "outDir": "../dist",
+        "removeComments": true,
+        "rootDir": "./",
         "sourceMap": true,
         "target": "es5",
```

Comment:
Unrelated and non-blocking Q for @janechu - Do you know if there is a reason we are targeting ES5 here instead of ES6 like all other packages?

Reply:
This is an old library we never got around to updating. I'm actually working on a major update for it now so it uses the foundation.

Reply:
party